packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -468,6 +476,21 @@ export class SpecBuilder {
     return this;
   }
 
+  /**
+   * Expects a document to be in limbo, but *without* a targetId.
+   */
+  expectInactiveLimboDocs(...keys: DocumentKey[]): this {
```

Comment:
Elsewhere you're calling these "enqueued" limbo docs. Let's be consistent throughout.

Reply:
Done.  Do you think it is okay to leave the `expectedLimboDocs()` method with its current name?  It does not explicitly call out the fact that it is referring to "active" limbo documents, but this is the typical case used in many places in the spec tests.  Moreover, the difference between "enqueued" and "inactive" only matters here in the spec tests for limbo resolution throttling.

Reply:
I can see this going two ways:
1. Just leave it as-is, which is pretty much fine
2. Make `expectedLimboDocs` check both active and inactive docs and make a third `expectedActiveLimboDocs` that just checks actives.

Personally, I think (2) is veering into overengineering territory so I'm OK with just leaving it.

Reply:
I tend to agree.  I'll leave it as-is, with `expectedLimboDocs()` being the name of the method.